homeassistant/components/lifx/light.py

```diff
@@ -567,15 +567,17 @@ def max_mireds(self):
         return math.ceil(color_util.color_temperature_kelvin_to_mired(kelvin))
 
     @property
-    def supported_features(self):
-        """Flag supported features."""
-        support = SUPPORT_BRIGHTNESS | SUPPORT_TRANSITION | SUPPORT_EFFECT
-
+    def color_mode(self) -> str:
+        """Return the color mode of the light."""
         bulb_features = lifx_features(self.bulb)
         if bulb_features["min_kelvin"] != bulb_features["max_kelvin"]:
-            support |= SUPPORT_COLOR_TEMP
+            return ColorMode.COLOR_TEMP
+        return ColorMode.BRIGHTNESS
 
-        return support
+    @property
+    def supported_color_modes(self) -> set[str] | None:
```

Comment:
```suggestion
    def supported_color_modes(self) -> set[ColorMode]:
```

Reply:
This recommendation might need https://github.com/home-assistant/core/pull/70806